src/components/breadcrumb/breadcrumb.stories.js

```diff
@@ -10,7 +10,6 @@ export default {
       description: {
         component: docsIntro({
           root: 'breadcrumb',
-          stylesheets: ['breadcrumb'],
           intro:
             'Use breadcrumbs to show the location of the current page within a navigational hierarchy. Breadcrumbs get arrow separators automatically via CSS.',
```

Comment:
Ik zou die "Breadcrumbs get arrow separators automatically via CSS." weglaten, is niet van toepassing bij ons.